packages/repco-core/src/datasources/activitypub.ts

```diff
@@ -591,9 +611,14 @@ export class ActivityPubDataSource
       subtitleEntities && entities.push(...subtitleEntities)
     }
 
+    var titleJson: { [k: string]: any } = {}
+    titleJson[video.language?.name || 'de'] = { value: video.name }
```

Comment:
- Ingo reported a bug about `English` appearing as an object key, where it should be `en`. I suspect this line being the cause.